spec/ttml1.xml

```diff
@@ -4598,22 +4598,22 @@ as a style property only to those element types indicated in the following table
 computed value of the style property must be considered to be no less than the
 size of the largest font size that applies to the element and its
 descendant elements in the intermediate synchronic document as determined by
-<specref ref="semantics-region-layout-step-1"/>.</p>
+<specref ref="semantics-region-layout-step-1"/>. However, the value <code>normal</code> is not relative,
+and is therefore inherited as-is, as specified at <specref ref="semantics-style-resolution-processing-css"/>.</p>
 <note role="elaboration">
-<p>It is recommended that presentation processors use a computed line height that is
```

Comment:
Please restore the original text, as this editorial change is gratuitous and unnecessary.

Reply:
The current text does not explicitly associate the note with the condition that "the value of this attribute is <code>normal</code>"

Reply:
Yes it does. That's what the "If ..." antecedent says. If = When.

Reply:
I think we have more of a problem than I initially suspected since the text here (and TTML2) makes normative statements about resolving ``normal`` to an absolute value in the defining context (rather than the usage context). We may need to introduce the category *used values* after all.